tests/5.0/taskloop/test_taskloop_lastprivate.c

```diff
@@ -0,0 +1,56 @@
+//-------------------------------- test_taskloop_lastprivate.c ------------------------------------------------//
+//
+// OpenMP API Version 5.0 August 2023
+//
+// This test checks the 'taskloop' directive with the 'lastprivate' clause specified.
+// The 'taskloop' construct parallelize loops with independent iterations by creating tasks. 
+// It allows for efficient parallel execution of loop iterations by distributing them among multiple threads. 
+// The 'lastprivate' clause ensures that the last value of variable in a loop is available after loop completion.  
+//-------------------------------------------------------------------------------------------------------------//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define NUM_THREADS 1000
+#define NUM_TASKS 1000 
+
+int test_taskloop_lastprivate() {
+    
+    int errors = 0;
+   
+    int val = 100;
+    int task_vals[NUM_TASKS];
+
+    int last_itr = NUM_TASKS - 1;
+    
+    #pragma omp parallel num_threads(NUM_THREADS)
+    {
+        #pragma omp single
+        {
+            #pragma omp taskloop lastprivate(val) //val is set to ZERO for each task
```

Comment:
```suggestion
            #pragma omp taskloop lastprivate(val) 
```
`//val is set to ZERO for each task`: val is not set to zero, it is undefined private value